src/components/pages/Login/LoginForm.js

```diff
@@ -22,7 +22,7 @@ const LoginForm = props => {
         style={{ height: '85vh' }}
         verticalAlign="middle"
       >
-        <Grid.Column style={{ maxWidth: 300 }}>
+        <Grid.Column style={{ maxWidth: 330 }}>
           {state.errors.length && (
             <Message error>
```

Comment:
Can we use `react-toastify` to render the success/error messages?

Reply:
I will look into it